sdk/src/pubkey.rs

```diff
@@ -99,8 +103,34 @@ impl Pubkey {
             hasher.hash(seed);
         }
         hasher.hashv(&[program_id.as_ref(), "ProgramDerivedAddress".as_ref()]);
+        let hash = hasher.result();
```

Comment:
@aeyakovenko I remove the extra hash of the result, think we should leave that in?

Used to be
```rust
let hash = hash(hasher.result().as_ref());
```